documentation/docs/02-template-syntax/08-bindings.md

```diff
@@ -58,6 +58,34 @@ On `<input>` elements with `type="file"`, you can use `bind:files` to get the [`
 <button onclick={clear}>clear</button>
 ```
 
+`FileList` objects also cannot be modified, so if you want to e.g. delete a single file from the list, you need to create a new `DataTransfer` object and add the files you want to keep.
+
+```svelte
+<script>
+	let files = $state();
+
+	function removeFile(file) {
+		const dataTransfer = new DataTransfer();
+		for (const oldFile of files)
+			if (oldFile !== file)
+				dataTransfer.items.add(oldFile);
+
+		files = dataTransfer.files;
+	}
+</script>
+
+<input bind:files multiple />
+
+{#each files as file (file)}
+	<div>
+		{file.name}
+		<button onclick={() => removeFile(file)}>Remove</button>
+	</div>
+{/each}
+```
```

Comment:
Is that code example necessary? It's taking up quite a bit of space, I think the "hey watch out for that" is enough, and then people can read up on how the `DataTransfer` API works (like I should have done, to not provoke this PR 😄 )

Reply:
Removed the code but kept the note above if that's what you meant.